package.json

```diff
@@ -13,7 +13,8 @@
   },
   "scripts": {
     "test": "mocha --require @babel/register",
-    "build": "node build.js"
+    "build": "node build.js",
+    "dev": "npm run build && http-server"
```

Comment:
- git clone
- npm install
- npm run dev

The examples are not working (except browser connect which does not use a bundler)

Reply:
I updated the code with rebase, you can retry.